web/src/views/FileBrowserView/FileBrowser.vue

```diff
@@ -4,11 +4,6 @@
     indeterminate
   />
   <PublishFileBrowser
-    v-else-if="DJANGO_API"
```

Comment:
The `v-if` above looks as it could also be simplified--is `girderDandiset` still a thing in this context?

Reply:
`girderDandiset` still exists in this PR but it's removed in #778. The `v-if` is simplified in that PR too.

Reply:
Heh just saw that in 778. Thanks!